src/cpp/rtps/reader/StatelessReader.cpp

```diff
@@ -252,13 +255,15 @@ bool StatelessReader::change_received(
 
             if (getListener() != nullptr)
             {
+                // WARNING! This method could destroy the change
                 getListener()->onNewCacheChangeAdded(this, change);
             }
 
             new_notification_cv_.notify_all();
 
             // statistics callback
-            on_subscribe_throughput(change->serializedPayload.length);
+            // std::cout << "[FAST STATELESS READER] on_subscribe_throughput sending thr" << std::endl;
```

Comment:
Seems this was left here unintentionally

Reply:
:+1: